client/src/hooks/useCountUp.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useRef, useState } from 'react';
+
+interface CountUpProps {
+  end: number;
+  start?: number;
+}
+
+const useCountUp = ({ start = 0, end }: CountUpProps) => {
```

Comment:
```suggestion
const useCountUp = ({ end, start = 0 }: CountUpProps) => {
```
위처럼 optional parameter 순서만 바꿔주시면 좋을 것 같습니다!